src/graphql/mutations/forgeSubmit.js

```diff
@@ -1,11 +1,21 @@
 const defaultResponseQuery = `{
   id
   eid
+  status
+  resolvedPayload
+  payload
   payloadValue
   currentStep
-  completedAt
+  totalSteps
+  continueURL
+  reviewData
+  completionPercentage
+  isExcluded
+  touchedByUser
+  requestMeta
   createdAt
   updatedAt
+  completedAt
```

Comment:
These are all the things that are possible. Do we want them all?

Reply:
Probably fine to keep unless there's a performance hit with always getting everything all the time?

Reply:
I'd keep 

`status, continueURL, completedAt`